user/advanced-topics/bind-dirs.md

```diff
@@ -44,9 +44,15 @@ Inside the app qube.
 
 4. Save.
 
-5. Reboot the app qube.
+5. If the directory you are trying to persist doesn't already exist in the app qube, you'll need to create the directory with its full path, under `/rw/bind-dirs`. In this case you would do:
+   ```
+   sudo mkdir -p /rw/bind-dirs/var/lib/tor
+   ```
+   This case happens only when the template on which this app qube is based does have this directory.
```

Comment:
Just to confirm, is it supposed to be "does have this directory" or "does not have this directory"?

Reply:
Good catch. I meant to say _does not_

Reply:
Fixed now via [eee99c7](https://github.com/QubesOS/qubes-doc/pull/1292/commits/eee99c70bb0dbce8a1a3de0b9c2b75ff72dbb8f5).